### DISCUSSION TITLE
ant-design 项目 tsconfig.json 是否开启 strict

### DISCUSSION THREAD

**Discussion by crazyair** [ant-design/ant-design#47991] [Ideas]
如果不开启，会很容易导出异常发生，如：
此处是开启后，会有错误提示，没开启是没有提示的
![image](https://github.com/ant-design/ant-design/assets/7971419/7ff127d3-7ebc-41e1-8233-eff4fde10218)
是因为这里 e 没有加  `?`，而实际 `onCopy` 类型定义是这样的
![image](https://github.com/ant-design/ant-design/assets/7971419/d2b8f8a1-0a85-411c-8185-703303ee7e61)

如果用户在 `onCopy` 使用了 e 的方法，就会导致报错

开启后，找到 `Found 199 errors in 118 files.` 处错误

**Comment by MadCcc** [comment]
rc 也都没有开启，这些 error 大概率修不完。
虽然没有打开 `strict`，但是有 `strictNullChecks`。

**Reply by crazyair** [reply]
node_modules 里的不会在 ant-design 报错，就像我们业务项目开启了，也能照常用  antd

**Comment by afc163** [comment]
开启吧。

**Reply by crazyair** [reply]
这个在 mater 改还是 feature

**Reply by afc163** [reply]
master
